
# This represents the type of strategy the pathplanning should follow.

# ucl definition of a desired pose message.

Header header

int16 type

float64 x
float64 y
float64 z
